### PR TITLE
Update message level from :info to :warning

### PR DESCRIPTION
**Pull request by tlikai**
#23

### CODE REVIEW

File: lib/pronto/rubocop.rb [hunk 1]
```diff
@@ -60,12 +60,7 @@ def processed_source_for(patch)
     end
 
     def level(severity)
-      case severity
-      when :refactor, :convention
-        :info
-      when :warning, :error, :fatal
```
Comment (mmozuras) [inline lib/pronto/rubocop.rb]: @tlikai I think that `:error` and `:fatal` should continue translating to `:error` and `:fatal`. What do you think?

**Comment by mmozuras** [comment]
@nfm @patrickdavey @garin @ernetas what do you think?

**Comment by patrickdavey** [comment]
Is there any way that this could be configurable? I think that'd be the ideal.

File: lib/pronto/rubocop.rb (hunk 1 above)
Comment (tlikai) [inline lib/pronto/rubocop.rb]: agree

**Comment by mmozuras** [comment]
🆗, let's change it to ⚠️. I agree that ideally it would be configurable via `.pronto.yml`.

**Comment by mmozuras** [comment]
Released as [v0.8.1](https://github.com/mmozuras/pronto-rubocop/releases/tag/v0.8.1).

**Pull request merged**